src/phylo/phylogeny.jl

```diff
@@ -0,0 +1,230 @@
+# phylo/phylogeny.jl
+# ==================
+#
+# Types and methods for phylogenetic trees.
+#
+# Part of the Bio.Phylo module.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+
+
+# The Phylogeny Type
+# -------------------
+
+"""
+Phylogeny represents a phylogenetic tree.
+
+Fields:
+
+    - graph: A LightGraphs directional graph.
```

Comment:
`LightGraphs` will not make sense to those who don't know the LightGraphs.jl package.